tests/testthat/test-set-credentials.R

```diff
@@ -0,0 +1,55 @@
+# Run this test for users to make sure the set credentials from R object features
+# of doAzureParallel are still working
+context("set credentials from R object scenario test")
+test_that("Basic scenario test", {
+  testthat::skip("Live test")
+  testthat::skip_on_travis()
+  
+  # set your credentials
+  credentials <- list(
+    "batchAccount" = list(
+      "name" = "batchaccountname",
+      "key" = "batchaccountkey",
+      "url" = "https://batchaccountname.region.batch.azure.com"
+    ),
+    "storageAccount" = list("name" = "storageaccountname",
+                            "key" = "storageaccountkey")
+  )
+  doAzureParallel::setCredentialsObject(credentials)
```

Comment:
Is this different than the setCredentials call? Can't we have an override?

Reply:
combined both methods to one setCredentials() function and use class() to check whether it's a json file or a R credentials object.